app/styles/_core.less

```diff
@@ -571,6 +571,46 @@ label.checkbox {
   }
 }
 
+.compute-resource {
+  .resource-size {
+    display: flex;
+    height: 27px;
+    .resource-amount {
+      width: 80%;
+      input {
+        width: 100%;
+      }
+    }
+    .resource-unit {
+      width: 20%;
+      .ui-select-container {
+        height: 27px;
+      }
+      .ui-select-choices,
+      .ui-select-toggle {
+        width: 100%;
+      }
+    }
+  }
+}
+
+@media (max-width: @screen-sm-max) {
+  .compute-resource {
+    .inline-select {
+      // Spacing below input at mobile sizes.
+      margin-top: 5px;
+    }
+    .resource-size {
+      .resource-amount {
+        width: 70%;
+      }
+      .resource-unit {
+        width: 30%;
```

Comment:
Interesting. For me there is plenty of room. Do we have a different font for ui-select on Linux?

![screen_shot_2016-12-19_at_11_43_18_am](https://cloud.githubusercontent.com/assets/1167259/21320884/85738070-c5e0-11e6-978c-a67e1bb82efe.png)